src/backend/utils/indexer.js

```diff
@@ -82,6 +82,7 @@ const search = async (textToSearch, filter = 'post') => {
         fields: createFieldsFromFilter(filter),
       },
     },
+    sort: filter === 'author' ? [{ published: { order: 'desc' } }] : undefined,
```

Comment:
Let's follow the same pattern as `createFieldsFromFilter` above, and move this to a separate function.

Reply:
ok, made the seperate function.